docs/dev/table/tableApi.md

```diff
@@ -328,12 +328,12 @@ orders = table_env.scan("Orders");
       <td>
         <p>Similar to a SQL SELECT statement. Performs a select operation.</p>
 {% highlight python %}
-orders = table_env.scan("Orders");
-result = orders.select("a, c as d");
+orders = table_env.scan("Orders")
```

Comment:
Please add Python `Examples` in `Overview & Examples` section.

Reply:
Done.